src/builders/browser/index.ts

```diff
@@ -43,13 +51,19 @@ export class SingleSpaBrowserBuilder extends BrowserBuilder {
 
     return webpackMerge.smart(config, {
       output: {
-        library: options.libraryName,
+        library: libraryName,
         libraryTarget: options.libraryTarget,
       },
       externals: {
         'zone.js': 'Zone',
       },
-    })
+    });
+  }
+
+  private getProjectName() {
+    if (this.context.targetSpecifier && this.context.targetSpecifier.project) {
+      return camelize(this.context.targetSpecifier.project);
```

Comment:
camelize is nice when the libraryName is a global variable in the DOM, but maybe confusing or not nice when it's not. For example, if you have a project called `angular-app`, it would be confusing if it went into SystemJS as `angularApp` instead of `angular-app`. I think I would prefer just leaving the project name alone for this reason, and letting people specify a custom name if they want one.